CMakeLists.txt

```
@@ -204,6 +205,10 @@ else()
     set(CMAKE_CXXFILT ${TOOLCHAIN_PREFIX}c++filt)
 endif()
 
+if(LINT_IWYU)
+    set(CMAKE_CXX_INCLUDE_WHAT_YOU_USE ${CMAKE_SOURCE_DIR}/Toolchain/Local/iwyu/${SERENITY_ARCH}/bin/include-what-you-use;-Xiwyu;--no_default_mappings)
```

This should really be a cache variable instead:
```suggestion
    set(CMAKE_CXX_INCLUDE_WHAT_YOU_USE
        "${CMAKE_SOURCE_DIR}/Toolchain/Local/iwyu/${SERENITY_ARCH}/bin/include-what-you-use"
        -Xiwyu
        --no_default_mappings
        CACHE STRING
        "Semicolon delimited command to launch IWYU with"
    )
```